lib/src/lifecycle_module.dart

```diff
@@ -59,22 +59,22 @@ abstract class LifecycleModule extends SimpleModule with Disposable {
   int _instanceId = _nextId++;
 
   List<LifecycleModule> _childModules = [];
-  Logger _logger;
-  String _defaultName;
-  LifecycleState _previousState;
-  LifecycleState _state = LifecycleState.instantiated;
-  Completer<Null> _transition;
-  Span _activeSpan;
+  late Logger _logger;
+  String? _defaultName;
+  LifecycleState? _previousState;
+  LifecycleState? _state = LifecycleState.instantiated;
+  Completer<Null>? _transition;
+  Span? _activeSpan;
 
   // Used by tracing to create a span if the consumer specifies when the module
   // reaches its first useful state
-  DateTime _startLoadTime;
+  DateTime? _startLoadTime;
 
   // Lifecycle event StreamControllers
-  StreamController<LifecycleModule> _willLoadChildModuleController =
-      StreamController<LifecycleModule>.broadcast();
-  StreamController<LifecycleModule> _didLoadChildModuleController =
-      StreamController<LifecycleModule>.broadcast();
+  StreamController<LifecycleModule?> _willLoadChildModuleController =
+      StreamController<LifecycleModule?>.broadcast();
+  StreamController<LifecycleModule?> _didLoadChildModuleController =
+      StreamController<LifecycleModule?>.broadcast();
```

Comment:
The other controllers below are `StreamController<LifecycleModule>` can we also make these 2 non-nullable if possible?

Reply:
Updated as suggested